LayoutTests/imported/w3c/web-platform-tests/scroll-animations/view-timelines/intermediate-transform.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html id="top">
+<meta charset="utf-8">
+<title>View timeline delay</title>
+<link rel="help" href="https://drafts.csswg.org/scroll-animations-1/#viewtimeline-interface">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/web-animations/testcommon.js"></script>
+<script src="/scroll-animations/scroll-timelines/testcommon.js"></script>
+<script src="/scroll-animations/view-timelines/testcommon.js"></script>
+<style>
+  #container {
+    border:  10px solid lightgray;
+    overflow-x: scroll;
+    height:  200px;
+    width: 200px;
+  }
+  #content {
+    rotate: 90deg;
+    width:  1800px;
+    margin: 0;
+  }
+  .spacer {
+    width:  800px;
+    display:  inline-block;
+  }
+  #target {
+    background-color:  green;
+    height:  100px;
+    width:  100px;
+    display:  inline-block;
+    font-size: 10px;
+  }
+</style>
+<body onload="runTests()">
+<div id="container">
+    <div id="content">
+      <div class="spacer"></div>
+      <div id="target"></div>
+      <div class="spacer"></div>
+    </div>
+  </div>
+</body>
+<script type="text/javascript">
+    function raf() {
+        return new Promise(resolve => {
+        requestAnimationFrame(() => {
+            requestAnimationFrame(resolve);
+        })
+        });
+    }
+    function runTests() {
+    promise_test(async t => {
+
+    const timeline = new ViewTimeline({
+        subject: target
+    });
+    const anim =
+        target.animate(
+            { backgroundColor: ['green', 'red' ] },
+            { duration: 100,
+            timeline: timeline });
+    await raf();
+    console.log(timeline.startOffset);
+    // assert_px_equals(timeline.startOffset, 600, 'startOffset');
+    // assert_px_equals(timeline.endOffset, 900, 'endOffset');
```

Comment:
Looks like the actual test assertions are disabled.

Reply:
Fixed this.